docs/docusaurus.config.ts

```diff
@@ -29,10 +29,6 @@ const config: Config = {
     locales: ['en'],
   },
 
-  future: {
-    experimental_faster: true,
-  },
```

Comment:
🥲 why

Reply:
It broke the build 😭

Reply:
https://github.com/mrousavy/nitro/pull/694